### DISCUSSION TITLE
chalk doesn't work on github actions log?

### DISCUSSION THREAD

**Discussion by gyhyfj** [google/zx#633] [General] [answered]
## Expected Behavior

Expect `console.log(chalk.green('success!'))` to be green on github actions log

## Actual Behavior

Output is gray as default, not green

## Steps to Reproduce the Problem

```js
#!/usr/bin/env zx
/* eslint-disable */
console.log(chalk.green(' success!'))
```
Then run this from script of package.json: `test:  zx ./test.mjs`

## Specifications

- Version: "zx": "^7.2.1"
- Platform: ubuntu of github actions

**Comment by andrewmclagan** [comment, marked as answer]
set the environment variable `FORCE_COLOR=3`
